contributing/patches/load_local_synthetic_data.patch

```diff
@@ -0,0 +1,227 @@
+diff --git a/apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResource.java b/apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResource.java
+index db11ae61..f73f85de 100644
```

Comment:
General question: how close will the local data load be when compared to the data in `prod-sbx`?

Reply:
As close as possible is the goal. These documents will cover loading the 30000 beneficiary records and their claims from the public synthetic rif files. It's likely that that will be slightly out of step with `prod-sbx`, but should be reasonably close.

A future set of work is to make it as easy as possible for anyone to load any data, allowing someone to tailor the data in their local instance as they see fit.